python_ta/reporters/node_printers.py

```diff
@@ -128,6 +128,65 @@ def render_missing_space_in_doctest(msg, _node, source_lines=None):
     yield from render_context(line + 1, line + 3, source_lines)
 
 
+def render_pep8_blank_lines(msg, _node, source_lines=None):
+    """Render a PEP8 blank lines message."""
+    if "blank line" not in msg.msg:
+        yield from render_generic(msg, _node, source_lines)
+    elif "expected 1 blank line" in msg.msg:
+        line = msg.line - 1
+        yield from render_context(line - 1, line + 1, source_lines)
+        body = source_lines[line]
+        indentation = len(body) - len(body.lstrip())
+        yield (
+            None,
+            slice(None, None),
+            LineType.ERROR,
+            body[:indentation] + '"""INSERT NEW BLANK LINE HERE"""' + " " * indentation,
+        )
+        yield from render_context(msg.line, msg.line + 2, source_lines)
+    elif "expected 2 blank lines" in msg.msg:
+        line = msg.line - 1
+        if "found 0" in msg.msg:
+            yield from render_context(line - 1, line + 1, source_lines)
+            yield from (
+                (
+                    None,
+                    slice(None, None),
+                    LineType.ERROR,
+                    '"""INSERT NEW BLANK LINE HERE"""',
+                )
+                for _ in range(0, 2)
+            )
+        else:
+            line -= 1
+            yield from render_context(line - 1, line + 1, source_lines)
+            yield (line + 1, slice(None, None), LineType.ERROR, " " * 32)
```

Comment:
You can make a helper like `render_blank_line` to pull out this repeated functionality.